otter/test/rest/test_application.py

```diff
@@ -545,14 +545,16 @@ def test_health_check_endpoint_calls_store_health_check(self):
         """
         And returns the stringified json blob
         """
+        jsonobj = {'blargh': 'boo', 'inside': {'key': 'boom'}}
+
         def health_check():
-            return succeed({'blargh': 'boo'})
+            return succeed(jsonobj)
 
         otter = Otter(self.mock_store, 'region', health_check)
         self.root = otter.app.resource()
 
         resp = self.assert_status_code(200)
-        self.assertEqual(resp, json.dumps({'blargh': 'boo'}))
+        self.assertEqual(resp, json.dumps(jsonobj, indent=4))
```

Comment:
Oh, actually, there may be room for a spurious error in the unit test, since the order of keys in a json object is not sorted by default.

You could either change the test to only have one key in the dictionary, or you could use sort_keys=True to the implementation and unit test. probably some other options too but that's what comes to me now.